### ISSUE TITLE
Rename package?

### ISSUE THREAD

**Issue by amdelamar** [embeddedkafka/embedded-kafka#232]
_I have been using this library since the original was deprecated, and it works great, thank you!_

Just wanted to ask if there is any plan to rename the package to match the project domain at all?

Its a little weird right now adding the dependency and having a different import statement for the classes.

```scala
"io.github.embeddedkafka" %% "embedded-kafka" % "2.5.0" % "test,it"
```

```scala
import net.manub.embeddedkafka._
```

**Comment by francescopellegrini** [comment]
Hi @amdelamar, thank you for your kind words!
We discussed about this topic a while ago, notably [here](https://github.com/manub/scalatest-embedded-kafka/issues/151) and [here](https://github.com/embeddedkafka/embedded-kafka/issues/3#issuecomment-441996003), and in the end our resolution was that BC was paramount.

Maybe we could evaluate writing a one-time migration script using Scalafix. Of course PRs are highly appreciated! ;-)

**Comment by francescopellegrini** [comment]
Just a small update: I'm working on the aforementioned Scalafix rule to help users migrate to the new package naming.

**Comment by francescopellegrini** [comment]
Fixed by #233
